packages/email-notification-dispatcher/src/handlers/tenants/handleTenantVerifiedAttributeAssigned.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  EmailNotificationMessagePayload,
+  generateId,
+  missingKafkaMessageDataError,
+  NotificationType,
+  fromTenantV2,
+  VerifiedTenantAttribute,
+} from "pagopa-interop-models";
+import {
+  eventMailTemplateType,
+  retrieveHTMLTemplate,
+  retrieveTenant,
+} from "../../services/utils.js";
+import {
+  TenantHandlerParams,
+  getRecipientsForTenants,
+  retrieveAttribute,
+} from "../handlerCommons.js";
+
+const notificationType: NotificationType =
+  "certifiedVerifiedAttributeAssignedRevokedToAssignee";
+
+export async function handleTenantVerifiedAttributeAssigned(
+  data: TenantHandlerParams
+): Promise<EmailNotificationMessagePayload[]> {
+  const {
+    tenantV2Msg,
+    attributeId,
+    readModelService,
+    logger,
+    templateService,
+    userService,
+    correlationId,
+  } = data;
+
+  if (!tenantV2Msg) {
+    throw missingKafkaMessageDataError(
+      "tenant",
+      "TenantVerifiedAttributeAssigned"
+    );
+  }
+
+  const tenant = fromTenantV2(tenantV2Msg);
+
+  const [htmlTemplate, attribute] = await Promise.all([
+    retrieveHTMLTemplate(
+      eventMailTemplateType.tenantVerifiedAttributeAssignedMailTemplate
+    ),
+    retrieveAttribute(attributeId, readModelService),
+  ]);
+
+  const targets = await getRecipientsForTenants({
+    tenants: [tenant],
+    notificationType,
+    readModelService,
+    userService,
+    logger,
+    includeTenantContactEmails: false,
+  });
+
+  if (targets.length === 0) {
+    logger.info(
+      `No targets found for tenant. Agreement ${tenant.id}, no emails to dispatch.`
+    );
+    return [];
+  }
+
+  const tenantAttribute = tenant.attributes.find(
+    (attribute): attribute is VerifiedTenantAttribute =>
+      attribute.id === attributeId
```

Comment:
I'd change this to
```
attr.type === tenantAttributeType.VERIFIED && attr.id === attributeId
```
as done by `retrieveTenantVerifiedAttribute` in `tenantService`

It's safer because, if the attribute with that id were, by some bug, not a verified attribute, we'd end in the ! case of line 73

without the extra condition, we'd go on with the attribute even if it's not verified (the `is` assertion is in itself unchecked) but I guess run into an error because `.verifiedBy` is undefined so we cannot access its `.length`